### DISCUSSION TITLE
How do I disable the date prefix on file creation? It goes against the standardized file naming scheme of the web framework.

### DISCUSSION THREAD

**Discussion by anoduck** [estruyf/vscode-front-matter#869] [Q&A] [answered]
I am not creating a blog, I am creating full-fledged website, and using a date prefix prepended to every file created in frontMatter will simply not work, and needs to be disabled for in my configuration. 

I found a part of the documentation that mentions using a [static slug](https://frontmatter.codes/docs/content-creation/placeholders#static-placeholders), but I don't see how this will have any effect on how frontMatter creates new files.

It is really irritating, so it would be nice if someone could shed some enlightenment on the subject.

**Comment by estruyf** [comment, marked as answer]
On the content folder or content type, you can set the `filePrefix` setting. In your case, you can set it to `null`.

https://frontmatter.codes/docs/content-creation/content-folders

**Reply by anoduck** [reply]
So simple, and beautifully implemented. Why didn't I think to do this? 

Thank You so much for the solution. FrontMatter is an incredible accomplishment, implemented intelligently. Thank you.
